src/stores/user.js

```diff
@@ -16,8 +17,10 @@ export const useUserStore = defineStore({
       const userRef = collection(firebaseStore, collections.user);
       const q = query(userRef,where('uid','==',uid));
       const querySnapshot = await getDocs(q);
+      const docId = querySnapshot.docs[0].id; // gets the document id
```

Comment:
Just store the entire query snapshot object rather than the doc id. That is why the property is called 'doc'

Reply:
Removed that line and placed it in the if statement